Dockerfile

```diff
@@ -14,6 +14,11 @@ RUN bootsnap precompile --gemfile .
 
 FROM $base_image
 
+# Install `mongodb-database-tools` for 'mongoexport'
+RUN wget -P /tmp/ https://fastdl.mongodb.org/tools/db/mongodb-database-tools-ubuntu1604-x86_64-100.7.2.deb && \
+    apt-get install -y /tmp/mongodb-database-tools-ubuntu1604-x86_64-100.7.2.deb && \
+    rm /tmp/mongodb-database-tools-ubuntu1604-x86_64-100.7.2.deb
```

Comment:
```suggestion
# TODO: remove this temporary MongoDB package once we no longer need mongoexport (once the migration to Postgres is done).
ARG mongo_package=mongodb-database-tools-ubuntu2204-x86_64-100.7.2.tgz
ARG mongo_package_repo=https://fastdl.mongodb.org/tools/db
WORKDIR /tmp
ENV DEBIAN_FRONTEND=noninteractive
RUN wget --progress=mega "${mongo_package}/${mongo_package_repo}" && \
    apt-get install -y --no-install-recommends "${mongo_package}" && \
    rm -fr /tmp/*
```

Reply:
Normally you'd add the apt repo so that future builds will pick up security patches and whatnot, but since this is just a temporary thing to add a tool for the migration and updating it isn't really an issue, I think it's a good call to just install the package directly 👍 (This way we don't slow down the build anything like as much, so hopefully happier developers.)

The suggestion has a bunch of changes for readability/maintainability/robustness:

- Use the package that's built for the base image's Ubuntu version (2204 not 1604).
- Use WORKDIR to change directory in the target filesystem.
- Use non-interactive settings for command-line tools (and use the wget progress setting that doesn't blow out the build logs).
- DRY up pathnames, URLs etc.

Reply:
Oh I'm assuming this doesn't need to work on arm64 cos it's only ever gonna run in prod, right? (and only for a couple of months so no chance of it running into any move to ARM there)